### PR TITLE
Update RFStockalike to new install structure

### PR DESCRIPTION
**Pull request by Raptor831**

### CODE REVIEW

**Comment by Olympic1** [comment]
Change `"spec_version" : 1,` to `"spec_version" : "v1.2",`

File: NetKAN/RFStockalike.netkan [hunk 1]
```diff
@@ -7,13 +7,16 @@
 	"license"      : "CC-BY-SA",
 	"install" : [
 		{
-		    "file"       : "RFStockalike/GameData/RealFuels",
+		    "file"       : "RFStockalike/GameData/RealFuels-Stockalike",
 		    "install_to" : "GameData"
+		},
+		{
+		    "file"       : "RFStockalike/GameData/RealPlume/RealPlume-Stockalike",
```
Comment (Olympic1) [inline NetKAN/RFStockalike.netkan]: `RealPlume-RFStockalike` - you forgot the RF

**Comment by Raptor831** [comment]
Did I really do that? :-/ Thanks for the catch.

**Comment by Raptor831** [comment]
Since this PR failed again, I went to make sure I didn't have any more silly errors in the file. My initial checks didn't come up with anything. I'm a bit perplexed because all I switched was the filepaths and the dependancies. I hope it's not something silly I'm just overlooking, but thanks for the help!

**Comment by Olympic1** [comment]
Runs fine locally and installed everything correctly.

**Comment by Raptor831** [comment]
:+1: Just checking.

**Comment by Olympic1** [comment]
trying a rebuild

**Comment by Olympic1** [comment]
Merging, because it will always fail

**Pull request merged**